common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_backhaul.yaml

```diff
@@ -125,6 +125,9 @@ cACTION_BACKHAUL_CLIENT_ASSOCIATED_NOTIFICATION:
   iface_mac: sMacAddr
```

Comment:
This commit message (cdbaca92af7d91485d92f7b05bc8b2fc7e768e62: tlvf: beerocks message backhaul: add association frame to CLIENT_ASSO…) is too long.
You can remove "beerocks message backhaul" and just write "tlvf:".